install_files/securedrop-app-code/debian/control

```diff
@@ -3,8 +3,13 @@ Section: web
 Priority: optional
 Maintainer: SecureDrop Team <securedrop@freedom.press>
 Homepage: https://securedrop.org
+Build-Depends: debhelper (>= 9), dh-python, python3-all, python3-setuptools, dh-virtualenv
+Standards-Version: 3.9.8
+X-Python3-Version: >= 3.5
 
 Package: securedrop-app-code
 Architecture: amd64
-Depends: python-pip,apparmor-utils,gnupg2,haveged,python,secure-delete,sqlite3,${dist:Depends},libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,libpython2.7-dev
+Conflicts: libapache2-mod-wsgi
+Replaces: libapache2-mod-wsgi
+Depends: ${python3:Depends},${misc:Depends},apache2,paxctld,apparmor-utils,gnupg2,haveged,secure-delete,sqlite3,${dist:Depends},libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config
```

Comment:
it seems here like the 2 extra requirements are `${python3:Depends}` and `${misc:Depends}`. Do we know what requirements these include? The reason I ask is that the current cron-apt script uses the security list to pull in packages: https://github.com/freedomofpress/securedrop/blob/a1c2e2abf0d03ecbacc7d44838fc275d5ba97cee/install_files/ansible-base/roles/common/files/5-security#L2 . If the dependencies are not in that list, the install will fail. I will test the upgrade scenario once I upload the builder image

Reply:
Those are standard `Python3` and if we add anything special based on debian packaging guides.